### ISSUE TITLE
Add simulation method to LR scheduler base class

### ISSUE THREAD

**Issue by ottonemo** [dnouri/skorch#266]
It would be nice to have a method that simulates the learning rate schedule for a given number of epochs which returns `(time, lr)`. These values can then be visualized and verified easily. It probably helps during hyper-parameter tuning as well.

**Comment by benjamin-work** [comment]
It could also facilitate unit testing and the visualizations may become part of the docs.
